src/js/tabs/osd.js

```diff
@@ -1740,6 +1766,22 @@ TABS.osd.initialize = function (callback) {
         $('.stats-container div.cf_tip').attr('title', i18n.getMessage('osdSectionHelpStats'));
         $('.warnings-container div.cf_tip').attr('title', i18n.getMessage('osdSectionHelpWarnings'));
 
+
+        let numberOfProfiles = OSD.getNumberOfProfiles();
+
+        // Header for the switches
+        let headerSwitches_e = $('.elements').find('.osd-profiles-header');
+        for (let profileNumber = numberOfProfiles; profileNumber > 0 ; profileNumber--) {
+            headerSwitches_e.prepend('<span class="profileOsdHeader">' + profileNumber + '</span>');
+        }
+
+        // Populate the profiles selector
+        let osdProfileSelector_e = $('.osdprofile-selector');
```

Comment:
@mikeller I can do it, but I'm not too sure what is the sense of this option. It will only show a mess of elements, merging the elements of one profile with the others.  The user can't enable more than one profile at a time.
What is the purpose of that?

Reply:
Yeah, point taken.